plyer/platforms/linux/wifi.py

```diff
@@ -17,7 +17,10 @@
 
 class LinuxWifi(Wifi):
     names = {}
-
+    
+    Popen(["nmcli", "-t","connection", "show", "--active"], stdout=PIPE, stderr=PIPE).
```

Comment:
You have a dot (`.`) at the end :)

Anyway, please don't pipe `stderr`, otherwise it's the same like `try: <stuff> except: pass` i.e. just eating any error the binary might raise. It's better to actually have something in the log than nothing. :)